src/e2e-test/resources/errorMessage.properties

```diff
@@ -0,0 +1,11 @@
+errorMessageTemporaryBucket=Bucket name can only contain lowercase letters, numbers, '.', '_', and '-'.
+errorMessageDataset=Required property 'dataset' has no value.
+errorMessageReference=Required property 'referenceName' has no value.
+errorMessageColor=#a40403
+errorMessageValidation=No errors found.
+errorMessageErrorFoundValidation=1 error found
```

Comment:
`1` is variable here, right? Can we do something like `COUNT` and replace it like `PROPERTY`

Reply:
Same with `error` or `errors` according to `COUNT`.

Reply:
Changed.